ak_vendor/templates/report_template.html

```diff
@@ -946,7 +946,7 @@ <h2 class="analysis__cvss__scorevalue" style="color: {{ analysis.risk.value_colo
             </table>
           </section>
 
-          {% if analysis.regulatory.owasp or analysis.regulatory.cwe or analysis.regulatory.mstg or analysis.regulatory.asvs or analysis.regulatory.pcidss or analysis.regulatory.hipaa or analysis.regulatory.gdpr %}
+          {% if analysis.regulatory.owasp or analysis.regulatory.cwe or analysis.regulatory.mstg or analysis.regulatory.masvs or analysis.regulatory.asvs or analysis.regulatory.pcidss or analysis.regulatory.hipaa or analysis.regulatory.gdpr %}
```

Comment:
@Praseetha-KR @subho007 Should we remove this condition? Is it even possible for any testcase to have no compliance in this long list of compliances? 😆

Reply:
@rajan262 on a second thought, let's go ahead without removing it. If we have to make all compliances optional, then this if-condition will become necessary. Let's wait until the compliance based reports are implemented then we might have a strong reasoning.